content/sensu-go/6.2/operations/deploy-sensu/generate-certificates.md

```diff
@@ -225,12 +225,11 @@ You should now have a set of files for use by Sensu agents:
 
 filename           | description                  | required on agent?  |
 -------------------|------------------------------|---------------------|
-`ca.pem`           | Trusted CA root certificate  | {{< check >}}       |
 `agent.pem`        | Backend server certificate   | {{< check >}}       |
 `agent-key.pem`    | Backend server private key   | {{< check >}}       |
```

Comment:
Should these be "Agent certificate" and "Agent private key" instead of `Backend` in the description column?